clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -94,51 +96,30 @@ async function initialize(context: vscode.ExtensionContext) {
     const message = `${FAIL_CREATE_GLOBAL_STORAGE_MSG}: ${getErrorMessage(
       error,
     )}`;
-    outputChannel.appendLine(message);
+    getOutputChannel().appendLine(message);
     throw Error(message);
   }
-  const maybeE4E = await getE4EAPI();
-  const maybeZowe = await Utils.getZoweExplorerAPI();
 
   languageClientService = new LanguageClientService(
-    outputChannel,
+    getOutputChannel(),
     context.globalStorageUri,
     {
       executeCommand: (command, args, next) => {
         if (command == "missing copybook") {
-          copyBooksDownloader.clearProfiles();
+          externalApis.clearProfiles();
         }
         next(command, args);
       },
     },
   );
 
-  const copyBooksDownloader = new CopybookDownloadService(
-    context.globalStorageUri,
-    maybeZowe && "api" in maybeZowe ? maybeZowe.api : undefined,
-    maybeE4E && "api" in maybeE4E ? maybeE4E.api : undefined,
-    outputChannel,
-    new DownloadDiagnosticsService(),
-    () => languageClientService.invalidateConfiguration(),
+  await initializeExternalAPIs(context.globalStorageUri, () =>
+    languageClientService.invalidateConfiguration(),
```

Comment:
This should not be needed anymore.
```suggestion
  await initializeExternalAPIs(
    context.globalStorageUri,
    languageClientService.invalidateConfiguration,
  );
```